<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayForm</class>
 <widget class="QWidget" name="DisplayForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="locationGroupBox">
     <property name="title">
      <string>Show Times</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="citiesLabel">
        <property name="text">
         <string>Chose times to show in tooltips:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="showFajrCheckBox">
          <property name="text">
           <string>Fajr</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showShroukCheckBox">
          <property name="text">
           <string>Sunrise</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showDuhrCheckBox">
          <property name="text">
           <string>Dhuhr</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="showAsrCheckBox">
          <property name="text">
           <string>Asr</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showMaghribCheckBox">
          <property name="text">
           <string>Maghrib</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showIshaCheckBox">
          <property name="text">
           <string>Isha'a</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="otherSettingsGroupBox">
     <property name="title">
      <string>Display Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="showNextRadioButton">
        <property name="text">
         <string>Show next prayer time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="countdownRadioButton">
        <property name="text">
         <string>Countdown to next prayer</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="ignoreHourCheckBox">
          <property name="text">
           <string>Don't show hour if it is zero</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="ignoreSecsCheckBox">
          <property name="text">
           <string>Ignore seconds when more than one hour is remaining</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="showElapsedCheckBox">
          <property name="toolTip">
           <string>Show until 10 minutes current prayer time</string>
          </property>
          <property name="text">
           <string>Show time from previous prayer</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="hlLayoutBackground">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>BackGround :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="hlLayoutPrev">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>previous prayer :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="hlLayoutText">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>BackGround text :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="hlLayoutNext">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>next prayer :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ampmCheckBox">
     <property name="text">
      <string>Use AM/PM clock format</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>countdownRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>ignoreHourCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>countdownRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>ignoreSecsCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>countdownRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>showElapsedCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
